relay-base-schema/src/metrics/name.rs

```diff
@@ -94,6 +94,11 @@ impl MetricName {
             .into_iter()
             .find(|namespace| maybe_namespace == namespace.as_str())
     }
+
+    /// Returns a reference to the `str` inside the [`MetricName`].
+    pub fn name_eq(&self, other: &str) -> bool {
+        *self.0 == *other
+    }
```

Comment:
Docs are false, but do we need this at all, there is already a PartialEq impl, maybe we just need another PartialEq impl?